src/field/mobile-app-ui/index.md

```diff
@@ -0,0 +1,157 @@
+   ![Mergin Maps mobile app Selecting workspace](./mobile-app-switching-workspace.jpg "Mergin Maps mobile app Selecting workspace")
+
+- **Manage Account** is a shortcut to the <DashboardShortLink /> where you can manage your projects, workspaces and subscriptions
+- **Sign out** to sign out or to switch to a different account
+- **Close account** can be used to delete your account within the <MobileAppNameShort />
+
+:::danger WARNING
+Be careful! If you delete your account, you will lose access to your <MainPlatformName /> projects both on the mobile device and on the server.
+:::
+
+
+## Working with a project
+A project is opened by tapping its name in the [Home](#home) or in the [Projects](#projects) tab. You have to download the project in order to be able to work with it.
+
+When a project is opened, you will see the map window. The map can be moved by dragging around and zoomed in/out by pinching open/close. Tapping on a feature displays the attributes form. 
+
+![A project opened in Mergin Maps mobile app](./mobile-app-project-opened.jpg "A project opened in Mergin Maps mobile app")
+
+### Current position and GPS info
+To see your current position and GPS info, you have to enable location services in your mobile device.
+
+Tapping the **location button** in the right bottom corner of the screen centres the map to your current position and keeps it centred while you are moving. If you move the map manually, the auto-centre mode turns off automatically.
+
+![Mergin Maps mobile app GPS accuracy](./mobile-app-center-to-current-position.jpg "Mergin Maps mobile app GPS accuracy") 
+
+In the left bottom corner of the screen, there is a **GPS** button that displays the GPS accuracy as reported by your mobile device. The colour of the button denotes the accuracy threshold set in [GPS settings](#gps-settings) in the <MobileAppNameShort />.
+
+Tapping the GPS accuracy button opens the GPS info panel:
+![GPS info panel](./mobile-app-gps-info.jpg "GPS info panel") 
+
+- **Source**: internal GPS of the mobile device or [external](../external_gps/) GPS receiver connected via Bluetooth
+- **Longitude, Latitude**: current position
+- **X, Y**: current position in project's coordinate reference system
+- **Horizontal** and **Vertical accuracy** of the GPS position
+- **Altitude**: ellipsoidal height if internal GPS is used. [External GPS](../external_gps/) devices usually return orthometric heights (ellipsoid with the geoid separation applied).
+- **Satellites (in use/view)**: number of satellites
+- **Speed**
+- **Last fix**: time of the last received GPS position
+- **GPS antenna height** that can be set in [GPS settings](#gps-settings)
+
+
+### Sync
+The **Sync** button can be used to synchronise changes during the field survey.
+
+If automatic synchronisation of changes is allowed in the [Settings](#streaming-mode-and-recording-settings), this button will indicate that synchronisation is in progress.
```

Comment:
This button indicates the sync in progress even if the automatic synchronisation is not turned on.

```suggestion
If automatic synchronisation of changes is allowed in the [Settings](#streaming-mode-and-recording-settings), this button will indicate that synchronisation is in progress.
```

Do we have an article about automatic synchronisation? This would be a nice place refer to it.